Source/WebKit/UIProcess/API/Cocoa/_WKApplicationManifest.mm

```diff
@@ -77,6 +77,22 @@
     };
 }
 
+static std::optional<WebCore::ApplicationManifest::Shortcut> makeVectorElement(const WebCore::ApplicationManifest::Shortcut*, id arrayElement)
+{
+    if (![arrayElement isKindOfClass:_WKApplicationManifestShortcut.class])
+        return std::nullopt;
+
```

Comment:
Nit - you can omit this early return, since `dynamic_objc_cast` will return `nil` below if the class check fails.

Reply:
Removed this early return.